src/rapport/03_resultaten.Rmd

```diff
@@ -114,34 +160,72 @@ benthos_design <- benthos_afdw_spatial %>%
 # Horvitz-Thomson type schatters
 benthos_design %>%
   group_by(jaar) %>%
-  summarize(mzb_tonnage = survey_total(
+  summarize(
+    n = n(),
+    mzb_tonnage = survey_total(
     x = tot_afdw / 1e6,
     na.rm = TRUE,
     vartype = "ci",
-    level = 0.95))
+    level = 0.95)) %>%
+  kable(digits = 1, caption = "(ref:schattingen)")
 
-# ofwel droge stof via zero-inflated lognormal distributie?
+```
+
+```{r eval=FALSE}
+# ofwel droge stof via hurdle gamma distributie?
 # cf examples in ?survey::svymle (voor censored lognormal)
 # censored kan eventueel ook een optie zijn: alle nullen beschouwen als
 # links-gecensureerde data met kleine cutoff
 # zie ook ?survey::svysurvreg
+
+# not working
```

Comment:
Toch een poging gedaan om de verdeling realistischer te maken? Cool!

Reply:
De reden dat ik dit gedaan heb (helaas werkt het niet) is omdat ik de data ook geanalyseerd heb volgens het MRP (multilevel regression & poststratification) concept en daar toch behoorlijk andere schattingen/betrouwbaarheidsintervallen bekom. Hierdoor vroeg ik me af of design-based regressieschatters beter overeenkomen met MRP. Want nu is het moeilijk advies te geven welke juist is.

Reply:
MRP zie bijvoorbeeld: https://bookdown.org/jl5522/MRP-case-studies/introduction-to-mrp.html